composer.json

```diff
@@ -20,7 +20,7 @@
         "sebastian/exporter": "^1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.0|^5.0"
+        "phpunit/phpunit": "^5.4"
```

Comment:
Honestly, I don't see any reason to keep BC (or to support older versions) in dev requirements.